addon/webextension/background/selectorLoader.js

```diff
@@ -0,0 +1,83 @@
+/* globals browser, catcher */
+window.selectorLoader = (function () {
+  const exports = {};
+  // These modules are loaded in order, and some need to be listed before others
+  // due to dependencies:
+  const scripts = [
+    "catcher.js",
+    "background/selectorLoader.js",
+    "selector/callBackground.js",
+    "clipboard.js",
+    "makeUuid.js",
+    "build/shot.js",
+    "randomString.js",
+    "domainFromUrl.js",
+    "build/inlineSelectionCss.js",
+    "selector/documentMetadata.js",
+    "selector/util.js",
+    "selector/ui.js",
+    "selector/shooter.js",
+    "selector/uicontrol.js"
+  ];
+
+  exports.unloadIfLoaded = function () {
+    return browser.tabs.executeScript({
+      code: "window.selectorLoader && window.selectorLoader.unloadModules()"
+    }).then(result => {
+      return result && result.toString() === "true";
```

Comment:
Ah, yeah. Looking at the [docs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/executeScript#Return_value) for executeScript, the returned value is what's returned by `unloadModules`...which should be the boolean `true`, not the string "true", but maybe that's a side effect of Promisification